aziotctl/Cargo.toml

```diff
@@ -27,6 +27,7 @@ sysinfo = "0.15.3"
 tokio = { version = "1", features = ["macros", "fs", "io-util", "rt-multi-thread"] }
 toml = "0.5"
 url = "2"
+users = "0.11"
```

Comment:
Use the `nix` crate that we already use.

Reply:
Thanks! I fixed it.